### DISCUSSION TITLE
Language bar not showing on my repository page

### DISCUSSION THREAD

**Discussion by Vaneeza-7** [github-linguist/linguist#6428] [Classification] [answered]
## Problem description
I have repositories containing sql files for which language bar is not showing. It's probably not detecting the language. Still, its applying syntax highlighting to the sql file itself.

### URL of the affected repository:
https://github.com/Vaneeza-7/SQL-UEFA-Champions-League-2016-2022
https://github.com/Vaneeza-7/Airport-Database

### Last modified on:
24-05-2023 (first repository)
23-05-2023 (second repository)

### NOTE:
For the first repository, I do not have a local repo linked with it, I have just uploaded files via GitHub UI.
For the second repository, I do have a local repo and most of the commits are via cmd.

**Comment by DecimalTurn** [comment, marked as answer]
In some cases, SQL statements can be used to represent data, so it's ignored by default when Linguist calculates language stats (see https://github.com/github-linguist/linguist/blob/master/docs/troubleshooting.md#my-repository-isnt-showing-my-language for more details)

If you want to override this behavior, you can add an [override](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md) to your `.gitattributes` file:

```
*.sql linguist-detectable
```
